x/concentrated-liquidity/swaps_test.go

```diff
@@ -1630,7 +1630,7 @@ var (
 			ExpectedSecondUpperTickSpreadRewardGrowth: secondPosition{TickIndex: 322500, ExpectedSpreadRewardGrowth: cl.EmptyCoins},
 			NewLowerPrice: osmomath.NewDec(5500),
 			NewUpperPrice: osmomath.NewDec(6250),
-			ExpectedSpreadRewardGrowthAccumulatorValue: osmomath.MustNewDecFromStr("0.007433904623597252"),
+			ExpectedSpreadRewardGrowthAccumulatorValue: osmomath.MustNewDecFromStr("0.007433904623597258"),
```

Comment:
What's the reason these changed?

Reply:
This PR changes rounding behavior is why

Reply:
Hm, this may be an invalid concern but I am worried since the rounding behavior changed in favor of the user.

Imagine a user's current expected spread rewards is 0.007433904623597252, and after the upgrade it is now 0.007433904623597258. Where do these additional funds come from? 

I guess the concern is, we calculated accum values with one rounding behavior, and now claim with a rounding behavior more in favor of the user, so what effect does this have (if any)?

Reply:
So remember this is only affecting the swap amount for what gets added to the spread factor accumulator. The reward calculation is based off the difference of two accumulator values.

So how we computed past accumulator values doesn't matter. It just matters that we are consistent with new additions